lib/parsers/log-parser.js

```diff
@@ -18,6 +18,12 @@ const ERROR_PATTERN = new RegExp('' +
   '(.+?)\\.?$'           // Message text, the ending period is optional for MiKTeX
 )
 
+// Pattern for obtaining the last macro in a line
+// Meant for catching the undefined control sequence
+const LAST_MACRO_PATTERN = new RegExp('' +
+  '(\\\\[^\\\\]+)$' // Macro name
```

Comment:
I'm not highly confident as to what ideally should a legal macro name in
this setting.

The problem is that the escape character, `\` by default, can be redefined, as well
as having anything being active letters (and thus treated as control sequences)

E.g. doing
```
\catcode`O=13\relax O
```
would make the letter O an unknown control sequence, and

```
\catcode`J=0\relax JUndefined
```
would make `JUnedfined` behave identically to `\Undefined` and this atom plugin
will report weird information.

You can't just look at the last space and go from there. If you consider
```
{}\some@undef@macro
```
Then the `{}` would be a part of the macro name.

However, I do argue that these cases are extraordinary circumstances,
and isn't really that necessary to implement.